### PR TITLE
Re-seed rand generator for parallel tracking processes

### PR DESCRIPTION
**Pull request by xresende**
multi-processes inherit random number generator states, thus introducing particle correlations in parallel tracking with quantum excitation turned on.

depends on 
https://github.com/lnls-fac/trackcpp/pull/67

### CODE REVIEW

File: pyaccel/tracking.py [hunk 1]
```diff
@@ -810,6 +819,13 @@ def find_m44(accelerator, indices='m44', energy_offset=0.0, fixed_point=None):
 
 # ------ Auxiliary methods -------
 
+
+def _set_random_seed():
+    # seed = _np.random.randint(0, 2**32-1)  # introduces correlation with parallel tracking!
+    seed = _random.randint(0, 2**32-1)  # c++ seed is a 32-bit unsigned int
+    _trackcpp.set_random_seed(seed)
```
Comment (fernandohds564) [inline pyaccel/tracking.py:826]: The problem with this solution is that we loose control over the repeatability of the random numbers we create.

Comment (xresende) [reply pyaccel/tracking.py:826]: yes. this needs changing for the final PR. I will add the solution we discussed offline in trackcpp.

Comment (xresende) [reply pyaccel/tracking.py]: done

Comment (fernandohds564) [reply pyaccel/tracking.py]: I think this problem will remain, even with the approach we discussed offline. Maybe we have to talk again to brainstorm about it.

Comment (fernandohds564) [reply pyaccel/tracking.py]: we decided to merge this one as is and address this issue of repeatability in future PRs.

**Review by fernandohds564 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by VitorSouzaLNLS — APPROVED** [review]

**Pull request merged**